app/api/dao/user_extension.py

```diff
@@ -97,20 +54,34 @@ def update_user_additional_info(data):
         timezone_value = data["timezone"]
         timezone = Timezone(timezone_value).name
         additional_info_data = {}
-        
-        existing_additional_info = UserExtensionModel.find_by_user_id(user_id)
-        if not existing_additional_info:
-            return messages.ADDITIONAL_INFORMATION_DOES_NOT_EXIST, HTTPStatus.NOT_FOUND
         try:
-            existing_additional_info.is_organization_rep = data["is_organization_rep"]
-            existing_additional_info.timezone = timezone
             additional_info_data["phone"] = data["phone"]
             additional_info_data["mobile"] = data["mobile"]
             additional_info_data["personal_website"] = data["personal_website"]
         except KeyError as e:
             return e, HTTPStatus.BAD_REQUEST
+        
+        existing_additional_info = UserExtensionModel.find_by_user_id(user_id)
```

Comment:
you might want to rename this to existing_user_extension because it is essentially that right? 
Because in the code later, when you do, existing_additional_info.additional_info it doesn't make sense.

Reply:
Done. I renamed this to useer_additional_info (for both existing and non-existing data)

Reply:
oh I meant if you do a find_by_user_id on a userextensionmodel what you'd expect is a user_extension in return. So rather than renaming the additional info you might want to rename this